app/components/mandaat/burgemeester-selector.js

```diff
@@ -0,0 +1,109 @@
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { BESTUURSORGAAN_CLASSIFICATIE_CODE_BURGEMEESTER } from 'frontend-lmb/utils/well-known-uris';
+import moment from 'moment';
+import {
+  getDraftStatus,
+  getEffectiefStatus,
+} from 'frontend-lmb/utils/get-mandataris-status';
+import { queryRecord } from 'frontend-lmb/utils/query-record';
+import { showErrorToast } from 'frontend-lmb/utils/toasts';
+
+export default class MandaatBurgemeesterSelectorComponent extends Component {
+  @service store;
+
+  @tracked persoon = null;
+  @tracked mandataris = null;
+
+  get bestuurseenheid() {
+    return this.args.bestuurseenheid;
+  }
+
+  get bestuursorgaanInTijd() {
+    return this.args.bestuursorgaanInTijd;
+  }
+
+  get bindingStart() {
+    return this.bestuursorgaanInTijd.bindingStart;
+  }
+
+  get bindingEinde() {
+    return this.bestuursorgaanInTijd.bindingEinde;
+  }
+
+  formatToDateString(dateTime) {
+    return dateTime ? moment(dateTime).format('YYYY-MM-DD') : undefined;
+  }
+
+  async getBurgemeesterMandaat() {
+    return await queryRecord(this.store, 'mandaat', {
+      filter: {
```

Comment:
This seems a bit of overkill, maybe best to simplify this after the routing of this page is improved.